spring4/src/test/java/org/jdbi/v3/spring4/TestJdbiTemplate.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jdbi.v3.spring4;
+
+import org.jdbi.v3.core.Jdbi;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.jdbc.datasource.embedded.EmbeddedDatabaseBuilder;
+import org.springframework.jdbc.datasource.embedded.EmbeddedDatabaseType;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
+import org.springframework.transaction.annotation.EnableTransactionManagement;
+
+import javax.sql.DataSource;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@RunWith(SpringJUnit4ClassRunner.class)
+@ContextConfiguration(classes = {TestJdbiTemplate.Config.class})
+public class TestJdbiTemplate {
+
+    @Autowired
+    private JdbiOperations jdbiOps;
+
+    @Test
+    public void testJdbiOps() {
+        assertThat(jdbiOps).isNotNull().isInstanceOf(JdbiTemplate.class);
+    }
+
+    @Configuration
+    @EnableTransactionManagement
```

Comment:
I believe to make `@EnableTransactionManagement` work you need to define a transaction manager in the configuration. In our case `DataSourceTransactionManager`:
```java
@Bean
public PlatformTransactionManager transactionManager(DataSource dataSource) {
    return new DataSourceTransactionManager(dataSource);
}
```

Reply:
maybe, will look that up. I am used to spring boot where the TM is defaulty defined as long as as a `PlatformTransactionManager` is on the classpath. 

then again the test suite is far from complete - this is WIP